src/backend/joanie/tests/test_api_course.py

```diff
@@ -156,6 +156,9 @@ def test_api_course_read_detail_anonymous(self):
                             "position": target_course.product_relations.get(
                                 product=product
                             ).position,
+                            "is_graded": target_course.product_relations.get(
```

Comment:
I think we should test that it is readonly by trying to update it from the API (same for the "position" field btw)

Reply:
I wrote a test to check that when the TargetCourseSerializer is valid, the validated_data dict is empty.
https://github.com/openfun/joanie/blob/2133bce21bf3fa0708bb1458341ce0e476d3ec8b/src/backend/joanie/tests/test_serializers_target_course.py

Reply:
This serializer is currently used by the CourseSerializer and we have already wrote tests to check that this kind of methods are not allowed on course api routes.

https://github.com/openfun/joanie/blob/2133bce21bf3fa0708bb1458341ce0e476d3ec8b/src/backend/joanie/tests/test_api_course.py#L649